packages/vue-query/src/__mocks__/useBaseQuery.ts

```diff
@@ -1,5 +1,8 @@
+import { vi } from 'vitest'
+
 const { useBaseQuery: originImpl, unrefQueryArgs: originalParse } =
-  jest.requireActual('../useBaseQuery')
+  // @ts-expect-error - vitest uses esmodules; tsconfig is not set to use them
+  (await vi.importActual('../useBaseQuery')) as any
```

Comment:
After reverting my change to the `tsconfig.json` this error shows up in the IDE via the ts language service, but running `pnpm test:ci` passes just fine even with the error marked in the IDE.

I've added this `// @ts-expect-error` so we don't see this via the IDE, but lmk if there's a preferred fix I'm not aware of.